### ISSUE TITLE
Ambiguous Argument new in 3.0 Release

### ISSUE THREAD

**Issue by bryanrite** [jdigger/git-process#145]
I've been trying out the 3.0.0 release, and get a new error. This is a long standing project that started on git-process 2.01.

When I try to `git sync`, or `pull-req`, I get the following error which did not occur on < 3.0:

```bash
$ git sync
git rev-list "-1" "--oneline" "origin/master..master"  2>&1:fatal: ambiguous argument 'origin/master..master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

If I checkout a local copy of master, then the command works fine:

```bash
$ git checkout master
Branch master set up to track remote branch master from origin.
Switched to a new branch 'master'

$ git checkout branch_test
Switched to branch 'branch_test'
Your branch is ahead of 'origin/master' by 1 commits.
  (use "git push" to publish your local commits)

$ git sync
You should remove your obsolete local branch, 'master'. Should I remove it for you? (Yn) |Y| n
(You can turn off this message using "git config gitProcess.keepLocalIntegrationBranch true").
Doing rebase-based sync
Fetching the latest changes from the server
Rebasing branch_test against origin/master
Fetching the latest changes from the server
Pushing to 'branch_test' on 'origin'.
```

*Edit:* If I then remove the local integration branch, the command will fail again.

In 3.0, are we supposed to keep a local integration branch or is there an issue of some sort?

Thanks for all your hard work!

**Comment by jdigger** [comment]
"are we supposed to keep a local integration branch or is there an issue of some sort?"

You definitely shouldn't need to keep a local integration branch. I'll see what might be going on.

**Comment by jdigger** [comment]
Looks to be related to GH-131

**Comment by jdigger** [comment]
This should be fixed in the just-released version 3.0.1

**Comment by bryanrite** [comment]
I can confirm, this seems to fix it for me.  Thanks @jdigger!
